db/migrate/20170915171451_add_status_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToItems < ActiveRecord::Migration[5.1]
+  def change
+    add_column :items, :status, :integer, default: 0
```

Comment:
im assuming the default is 0 for active and 1 for retired?
@Boots2291

Reply:
Yeah, missed a few files since I didn't want to add them all with `git add .`

I've included the changes to the model now, on there you can see the enum.